core/ustring.cpp

```diff
@@ -3194,6 +3194,50 @@ String String::simplify_path() const {
 	return drive + s;
 }
 
+String String::final_path_from_relative(const String p_relative) const {
```

Comment:
I'm not sure this method is needed at all, `DirAccess` supports relative paths too.

Reply:
Is there a method that does that within the String class? If there isn't should I leave it there as a utility function or should I remove it?

Reply:
I would say remove it for now, I prefer that we don't add new methods to String if we don't have a real need for them.

Reply:
Function removed from the string class.